package.json

```diff
@@ -81,7 +81,7 @@
     "moment-strftime": "~0.5.0",
     "moment-timezone": "~0.5.40",
     "numeral": "~2.0.6",
-    "patternfly": "~3.59.1",
+    "patternfly": "~3.59.5",
     "patternfly-bootstrap-treeview": "~2.1.7",
```

Comment:
I think we can drop this line possibly, since patternfly already depends on it, and they are on 2.1.10? See also patternfly itself: https://github.com/patternfly/patternfly-3/commit/63a20c8c656dd883b3c6249f21249e6e7bc30621

Reply:
Agreed, removing now